### ISSUE TITLE
Increase chunks

### ISSUE THREAD

**Issue by Sasamous** [nicolasjinchereau/acce1er8or#5]
Hello guys, I saw max chunks are 8
https://github.com/nicolasjinchereau/acce1er8or/blob/7492a608daa9548f38b403d22abca671f4095f2b/desktop-client/Download.cs#L40
I'm wondering, what about increasing it to 16?

There is the problem that some servers could not allow 16 threads? Maybe could be solved by a check like this:
https://github.com/nicolasjinchereau/acce1er8or/blob/7492a608daa9548f38b403d22abca671f4095f2b/desktop-client/Download.cs#L320-L329

**Comment by nicolasjinchereau** [comment]
I agree that the way chunks are handled is very primitive, but unfortunately, that part of the code was written in a rush, and will take significant effort to update and fix. A lot of things are hard-coded or mixed in with UI code, so it may be a long time before I can put in any work on this.

I've wanted to do this for a while, for the purpose of redistributing download tasks when some chunks finish early, but I can't promise anything will be done soon.
